testkit/tests/currency/cryptocurrency.rs

```diff
@@ -126,12 +126,13 @@ impl Transaction for TxCreateWallet {
     }
 
     /// Apply logic to the storage when executing the transaction.
-    fn execute(&self, view: &mut Fork) {
+    fn execute(&self, view: &mut Fork) -> ExecutionResult {
         let mut schema = CurrencySchema { view };
         if schema.wallet(self.pub_key()).is_none() {
             let wallet = Wallet::new(self.pub_key(), self.name(), INIT_BALANCE);
-            schema.wallets_mut().put(self.pub_key(), wallet)
+            schema.wallets_mut().put(self.pub_key(), wallet);
         }
```

Comment:
You could add an erroneous status signaling "wallet already exists" here. OTOH, when/if the `cryptocurrency` repo is moved into this repo, this can be done right there 😄

Reply:
I have created the issue for that: https://jira.bf.local/browse/ECR-682